packages/gas-fee-controller/src/GasFeeController.ts

```diff
@@ -348,9 +347,10 @@ export class GasFeeController extends StaticIntervalPollingController<
     this.getCurrentAccountEIP1559Compatibility =
       getCurrentAccountEIP1559Compatibility;
     this.#getProvider = getProvider;
-    this.EIP1559APIEndpoint = EIP1559APIEndpoint;
-    this.legacyAPIEndpoint = legacyAPIEndpoint;
+    this.EIP1559APIEndpoint = `${GAS_API_BASE_URL}/networks/<chain_id>/suggestedGasFees`;
+    this.legacyAPIEndpoint = `${GAS_API_BASE_URL}/networks/<chain_id>/gasPrices`;
     this.clientId = clientId;
+    this.infuraAuthToken = buildInfuraAuthToken(infuraAPIKey);
```

Comment:
Since the auth token is now derived from a single value, would it be cleaner to isolate the knowledge of the token inside the `gas-utils` and just pass the API key itself all the way to the util method?

Reply:
That make sense, I need to read the final PR once we do final changes :) 
https://github.com/MetaMask/core/pull/4068/commits/cd6bb24d1fb5e5215ff46f85b3e820c1064a910e